packages/dataviews/CHANGELOG.automattic.md

```diff
@@ -2,6 +2,12 @@
 
 ## Next 
 
+- Expose self-sufficient DataViews subcomponents:
+	- `<DataViews.Search />`
+	- `<DataViews.Filters />`
+	- `<DataViews.ViewConfig />`
```

Comment:
It feels like `ViewConfig` should be just `DataviewsViewConfigDropdown`

and `ViewTypeMenu` should be exposed separately rather than exporting the component wrapping both.

Reply:
Good one. In fact, I suffered from that limitation in the story

Reply:
According to https://github.com/WordPress/gutenberg/issues/63646, should we rename `ViewTypeMenu` to `LayoutSwitcher `?

Reply:
renamed here https://github.com/Automattic/wp-calypso/pull/103070/commits/829ede1cf0298d9efaba401b4c73a696f3e0d32a. I like `LayoutSwitcher` more :-)